pytissueoptics/rayscattering/opencl/CLParameters.py

```diff
@@ -0,0 +1,73 @@
+from pytissueoptics.rayscattering.opencl.CLProgram import CLProgram
+import numpy as np
+
+
+class CLParameters:
+    dataPointSize = 16
+    photonSize = 64
+    seedSize = 4
+    materialSize = 32
```

Comment:
These have changed in other branches and can change in the future. Hard to maintain here. We should consider to try getting this info from the CLObject itself.

Reply:
yes